linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/VarGen.scala

```diff
@@ -39,6 +39,8 @@ private[emitter] final class VarGen(jsGen: JSGen, nameGen: NameGen,
   import jsGen._
   import nameGen._
 
+  private implicit val globalRefTracking = config.topLevelGlobalRefTracking
```

Comment:
Add a comment why this is OK? (IIUC we only ever generate dangerous globalRefs in here).

Is there a fundamental reason not to wire this through with implicits? I realize it is more boilerplate-y in this case, but would be IMO less surprising?

Reply:
Huh good point. Probably I did that as a temporary measure while refactoring. I switched to implicits in individual methods.